<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STIX Document Validator</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="h_layout_main">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tab_widget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_files">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Documents</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="group_files">
            <property name="title">
             <string>Validate Files</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="FilesTableView" name="table_files">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::DropOnly</enum>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="v_layout_right">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Actions</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0" colspan="2">
                <widget class="QPushButton" name="btn_validate">
                 <property name="text">
                  <string>Validate</string>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QPushButton" name="btn_clear">
                 <property name="text">
                  <string>Clear Files</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="group_options">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>190</width>
                <height>200</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Options</string>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::FieldsStayAtSizeHint</enum>
               </property>
               <item row="0" column="0" colspan="2">
                <widget class="QCheckBox" name="check_best_practices">
                 <property name="text">
                  <string>Validate Best Practices</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QCheckBox" name="check_profile">
                 <property name="text">
                  <string>Valdiate STIX Profile</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_remove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_stix_validator">
    <property name="title">
     <string>STIX Validator</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_add_file"/>
   </widget>
   <widget class="QMenu" name="menu_options">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menu_xml_schema">
     <property name="title">
      <string>XML Schema</string>
     </property>
     <addaction name="action_set_schema_dir"/>
    </widget>
    <widget class="QMenu" name="menu_stix_profile">
     <property name="title">
      <string>STIX Profile</string>
     </property>
     <addaction name="action_set_stix_profile"/>
    </widget>
    <addaction name="menu_xml_schema"/>
    <addaction name="menu_stix_profile"/>
   </widget>
   <addaction name="menu_stix_validator"/>
   <addaction name="menu_file"/>
   <addaction name="menu_options"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_add_file">
   <property name="text">
    <string>Add Files...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_add_directory">
   <property name="text">
    <string>Add Directory...</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="action_set_schema_dir">
   <property name="text">
    <string>Set Schema Diectory</string>
   </property>
  </action>
  <action name="actionEnable">
   <property name="text">
    <string>Enable</string>
   </property>
  </action>
  <action name="action_set_stix_profile">
   <property name="text">
    <string>Set STIX Profile</string>
   </property>
  </action>
  <action name="action_enable_all_profile">
   <property name="text">
    <string>Enable All</string>
   </property>
  </action>
  <action name="action_disable_all_profile">
   <property name="text">
    <string>Disable All</string>
   </property>
  </action>
  <action name="action_enable_all_best_practice">
   <property name="text">
    <string>Enable All</string>
   </property>
  </action>
  <action name="action_disable_all_best_practice">
   <property name="text">
    <string>Disable All</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilesTableView</class>
   <extends>QTableView</extends>
   <header>cutiestix/widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
